### DISCUSSION TITLE
Streaming feedback

### DISCUSSION THREAD

**Discussion by rustatian** [roadrunner-server/roadrunner#1070] [Show and tell]
In this discussion, you may share your feedback about the new RR feature - HTTP response streaming.

**Comment by lanphan** [comment]
Hello @rustatian ,

In case there are so many request - response between 2 RR servers, is it possible to convert it to streaming and hope that it'll better for these 2 servers?

**Reply by rustatian** [reply]
Hey @lanphan, yep, that makes sense. Because if you have a lot of requests, a lot of time spends on the TCP handshake, connect, get the worker, etc. If that's possible to join them into the one request with a streaming response I guess you'll get a perf boost in your application. But, keep in mind, at the moment, RR supports streaming **responses** only.
